src/uct/ib/base/ib_device.c

```diff
@@ -298,6 +298,10 @@ static void uct_ib_async_event_handler(int fd, void *arg)
         level = UCS_LOG_LEVEL_DEBUG;
         break;
     case IBV_EVENT_DEVICE_FATAL:
+        ucs_assert(event.element.port_num == 0);
```

Comment:
why needed?
maybe just pass 0 instead of event.element.port_num?

Reply:
done